pkg/operator/controllers/imageconfig/image_controller.go

```diff
@@ -119,18 +119,26 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 // Switch case to ensure the correct registries are added depending on the cloud environment (Gov or Public cloud)
 func getCloudAwareRegistries(instance *arov1alpha1.Cluster) ([]string, error) {
-	var requiredRegistries []string
+	const replicationRegistryFormat = "%s.%s.data.%s"
```

Comment:
I'm not really a fan of this approach.  Would like to get other's thoughts but it's a bit more verbose using `fmt.Printf("%s.%s.data.%s", x, y, z)`

It's better though since we're defining it in one place using it in multiple though.  +1 on the DRY

Reply:
It seems we can still have the string inline, Uber's style guide recommendations seem to be more along the lines of "_if_ you're using a separately-declared format string, make it a const so `go vet` can do its thing".

Really I just wish Go had template strings.

Reply:
> It's better though since we're defining it in one place using it in multiple

From the changes in this PR it looks like we're defining `const replicationRegistryFormat = "%s.%s.data.%s"` twice (once here, once in the test) and subsequently only using it once in each place, so I don't think having it in-line would be too repetitive.